node_modules/col-canvas/lib/api.js

```diff
@@ -440,6 +440,54 @@ var deleteFile = module.exports.deleteFile = function(ctx, file, callback) {
   });
 };
 
+/* Conversations */
+
+/**
+ * Post to a Canvas conversation
+ *
+ * @param  {Context}      ctx                  Standard context containing the current user and the current course
+ * @param  {Number}       recipientId          The Canvas user ID of the recipient
+ * @param  {String}       subject              The subject of the conversation message
+ * @param  {String}       body                 The body of the conversation message
+ * @param  {Function}     callback             Standard callback function
+ * @param  {Object}       callback.err         An error that occurred, if any
+ * @param  {Object}       callback.response    The Canvas API response
+ */
+var postConversation = module.exports.postConversation = function(ctx, recipientId, subject, body, callback) {
+  log.debug({'recipientId': recipientId, 'subject': subject, 'body': body}, 'Posting to a Canvas conversation');
+
+  var url = util.format('%s/api/v1/conversations', getCanvasBaseURI(ctx.course.canvas));
+  var opts = {
+    'url': url,
+    'method': 'POST',
+    'headers': {
+      'Authorization': util.format('Bearer %s', ctx.course.canvas.api_key)
+    },
+    'body': {
+      'as_user_id': ctx.user.canvas_user_id,
+      'recipients': [ recipientId ],
+      'subject': subject,
+      'body': body,
+    },
+    'json': true
+  };
+  request(opts, function(err, response, body) {
+    if (err) {
+      log.error({'err': err, 'course': ctx.course.id, 'opts': opts}, 'Failed to post to a Canvas conversation');
+      return callback({'code': 500, 'msg': 'Failed to post to a Canvas conversation'});
+    } else if (response.statusCode !== 201) {
+      log.error({
+        'statusCode': response.statusCode,
+        'responseBody': response.bdody,
```

Comment:
@paulkerschen I like "bdody".  Is that what the kids are saying or is this a typo?